<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signin_GUI</class>
 <widget class="QDialog" name="Signin_GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin Sign In</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_topImg">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="myImg.qrc">:/myImg/topImg.jpg</pixmap>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_user">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>User:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_user">
       <property name="maxLength">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_password">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_password">
       <property name="maxLength">
        <number>20</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_exit">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pushButton_signin">
     <property name="text">
      <string>Sign In</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_user</tabstop>
  <tabstop>lineEdit_password</tabstop>
  <tabstop>pushButton_signin</tabstop>
  <tabstop>pushButton_exit</tabstop>
 </tabstops>
 <resources>
  <include location="myImg.qrc"/>
 </resources>
 <connections/>
</ui>
